README.md

```diff
@@ -1 +1,111 @@
-# covid-tracker
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Covid Tracker
+
+## About
+
+This is a covid tracker app where you can find latest information about COVID in Brazil, total number of cases, today new confirmed cases and deaths and more, you can also search by state and check specific information about any state of Brazil.
+
+Our development flow included: performing git flow practices & using components with routers, life cycle methods,handling events and state management by Redux store,actions, reducers in [duck Patern](https://github.com/erikras/ducks-modular-redux) with unit and component tests.
+
+## Built With
+
+- Javascript
+- React & Redux
+- Redux Middleware (Thunk, Logger)
+- Bootstrap
+- Jest for testing
+
+## Live Demo
+
+[Live Demo Link](https://blissful-northcutt-e13aab.netlify.app/)
+
+## Presentation video
+
+[Watch video](https://www.loom.com/share/bff86c2832e348889c00439235d390d7)
```

Comment:
- [ ] In your video presentation, you failed to highlight **the piece of code** you find interesting or are proud of making. Kindly remake your video and add this section.

  ![jonathas-code](https://user-images.githubusercontent.com/36542986/152297154-9bcc973e-ef24-4703-baf8-1db5cdd381e4.jpg)

Reply:
Thank you for your review! I just implemented the requested changes!